scribble-doc/scriblib/scribblings/autobib.scrbl

```diff
@@ -90,6 +92,29 @@ section for the bibliography. It has the contract
 (->* () (#:tag string? #:sec-title string?) part?)
 ]
 
+If provided, the function bound to @racket[cite-author]
+generates an element containing the authors of a paper.
+
+@racketblock[
+ (->* (bib?) element?)
+]
+
+If provided, the function bound to @racket[cite-year]
+generates an element containing the years the paper was
```

Comment:
Yup, in the case of multiple papers. I'll change that sentence to make it more clear.